src/main/java/com/ericsson/ei/subscription/SubscriptionHandler.java

```diff
@@ -94,35 +98,41 @@ public void checkSubscriptionForObject(final String aggregatedObject, final Stri
      * @param subscriptionData
      * @param id
      */
-    private void extractConditions(String aggregatedObject, String subscriptionData, String id) {
-        try {
-            JsonNode subscriptionJson = new ObjectMapper().readTree(subscriptionData);
-            LOGGER.debug("SubscriptionJson : " + subscriptionJson.toString());
-            LOGGER.debug("Aggregated Object : " + aggregatedObject);
-            ArrayNode requirementNode = (ArrayNode) subscriptionJson.get("requirements");
-            LOGGER.debug("Requirements : " + requirementNode.toString());
-            Iterator<JsonNode> requirementIterator = requirementNode.elements();
-            if (runSubscription.runSubscriptionOnObject(aggregatedObject, requirementIterator, subscriptionJson, id)) {
-                LOGGER.debug("The subscription conditions match for the aggregatedObject");
-                informSubscriber.informSubscriber(aggregatedObject, subscriptionJson);
-            }
-        } catch (Exception e) {
-            String msg = "Subscription: " + subscriptionData + "failed for ";
-            msg += "aggregated object: " + aggregatedObject;
-            LOGGER.error(msg, e);
-        }
+    private void extractConditions(String aggregatedObject,
+	    String subscriptionData, String id) {
+	try {
+	    JsonNode subscriptionJson = new ObjectMapper()
+		    .readTree(subscriptionData);
+	    LOGGER.debug("SubscriptionJson : " + subscriptionJson.toString());
+	    LOGGER.debug("Aggregated Object : " + aggregatedObject);
+	    ArrayNode requirementNode = (ArrayNode) subscriptionJson
+		    .get("requirements");
+	    LOGGER.debug("Requirements : " + requirementNode.toString());
+	    Iterator<JsonNode> requirementIterator = requirementNode.elements();
+	    if (runSubscription.runSubscriptionOnObject(aggregatedObject,
+		    requirementIterator, subscriptionJson, id)) {
+		LOGGER.debug(
+			"The subscription conditions match for the aggregatedObject");
+		informSubscriber.informSubscriber(aggregatedObject,
+			subscriptionJson);
+	    }
+	} catch (Exception e) {
+	    LOGGER.error("Subscription: {}, failed for aggregated object: {}",
+		    subscriptionData, aggregatedObject, e);
+	}
     }
 
     /**
-     * This method is responsible for displaying configurable application parameters
-     * like Subscription database name and collection name, etc.
+     * This method is responsible for displaying configurable application
+     * parameters like Subscription database name and collection name, etc.
      */
     @PostConstruct
     public void print() {
-        LOGGER.debug("SubscriptionDataBaseName : " + subscriptionDataBaseName);
-        LOGGER.debug("SubscriptionCollectionName : " + subscriptionCollectionName);
-        LOGGER.debug("MongoDBHandler object : " + mongoDBHandler);
-        LOGGER.debug("JmesPathInterface : " + jmespath);
+	LOGGER.debug("SubscriptionDataBaseName : {}", subscriptionDataBaseName);
```

Comment:
This PostConstruct may be removed, logging of those settings should be from ConfigLogger.java

Reply:
The database and collection name are logged in ConfigurationLogger but the other objects are not. I'll remove it.